tools/c7n_gcp/c7n_gcp/query.py

```diff
@@ -161,13 +167,57 @@ def resources(self, query=None):
                     local_session(self.session_factory).get_default_project())
                 return []
             raise
-        self._cache.save(key, resources)
-        return self.filter_resources(resources)
 
     def augment(self, resources):
         return resources
 
 
+class ChildResourceManager(QueryResourceManager):
+
+    @classmethod
+    def get_parent_annotation_key(cls):
+        parent_resource_type = cls.resource_type.parent[0]
```

Comment:
rename parent on the resource_type to parent_spec, also per previous comment, use a dictionary with named components instead of numeric offsets into a tuple.

Reply:
Done. Thanks